docs/streams/upgrade-guide.html

```diff
@@ -72,11 +72,16 @@ <h1>Upgrade Guide and API Changes</h1>
         More details about the new config <code>StreamsConfig#TOPOLOGY_OPTIMIZATION</code> can be found in <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-295%3A+Add+Streams+Configuration+Allowing+for+Optional+Topology+Optimization">KIP-295</a>.
     </p>
 
-    <h3><a id="streams_api_changes_250" href="#streams_api_changes_250">Streams API changes in 2.5.0</a></h3>
-    <p>
-        As of 2.5.0 Kafka we deprecated <code>UsePreviousTimeOnInvalidTimestamp</code> and replaced it with <code>UsePartitionTimeOnInvalidTimeStamp</code> as per
-        <a href="https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=130028807">KIP-530</a>
-    </p>
+    <h3><a id="streams_api_changes_250" href="#streams_api_changes_250">Streams API changes in 2.5.0</a>
+        <p>
+            We have added <code>CogroupedKStream</code>, which can be used to aggregate <code>KGroupStreams</code> into a <code>KTable</code>.
```

Comment:
We should primarily mention `cogroup()` operator, because this is method people will use. (also include link to the KIP wiki page)
```
We add a new <code>cogroup()</code> operator (via <link-to-wiki>KIP-150<link>)
that allows to aggregate multiple streams in a single operation.
Cogrouped streams can also be windowed before they are aggregated.
We refer to the <link>developer guide<link> for more details.
```

Reply:
that makes sense I was not really sure the purpose of this page